### DISCUSSION TITLE
How can I use the stat format with Gitlab ?

### DISCUSSION THREAD

**Discussion by gomme600** [priv-kweihmann/meta-sca#11292] [Q&A]
I came across this in the meta-sca documentation :

This export format is structured-acceptance-test exports and can be used in several editions of Gitlab. The complete format is described [here](https://github.com/fulldecent/structured-acceptance-test/blob/master/Stat-Output.md)

However, I can't seem to find any documentation on how to use this format within gitlab-ci. I only found [this](https://docs.gitlab.com/ee/ci/unit_test_reports.html).

Do you have any further explanation on how to integrate meta-sca results in gitlab-ci ?

Thanks

**Comment by priv-kweihmann** [comment]
https://github.com/priv-kweihmann/meta-sca/issues/5734 should provide you the answers you were looking for.
Please have a look at https://docs.gitlab.com/ee/user/project/merge_requests/code_quality.html#implementing-a-custom-tool (as pointed out in the original ticket).

**Comment by gomme600** [comment]
Thanks ! So I should be using codeclimate format instead of stat format for gitlab-ci ?

**Comment by priv-kweihmann** [comment]
A very good question - I can't give you a proper answer on that one, as I'm not having a suitable GitLab instance up and running at the moment - last time I checked codeclimate was the better choice though

**Comment by gomme600** [comment]
Ok ! I'll give codeclimate a try !
